### ISSUE TITLE
[RFC] Add Emoji in page titles, for easier visual recognition.

### ISSUE THREAD

**Issue by bobdenotter** [bolt/bolt#7024]
Does your browser toolbar often look like this, where you have a bunch of tabls open with the same Bolt icon? 

![screen shot 2017-09-25 at 17 13 14](https://user-images.githubusercontent.com/1833361/30816293-e5f12f12-a215-11e7-9eaa-b6d403d26008.png)

It gets hard to quickly recognize what is what: 

![screen_shot_2017-09-25_at_17_13_14](https://user-images.githubusercontent.com/1833361/30816504-6f179f38-a216-11e7-8ede-74c438f1ebf7.png)

My suggestion is to add emoji to the `<title>` tags of backend pages as well as docs, to make it easier to spot which tab is which. Something like this: 

![screen shot 2017-09-25 at 17 19 33](https://user-images.githubusercontent.com/1833361/30816572-9a1c0d18-a216-11e7-819c-49f4918ebbbd.png)
(please click the screenshot, to see it full size)

Note: I'm suggesting Emoji, because we can't use FontAwesome icons, like we do in the Bolt Interface. Emoji work well across all platforms, in all modern browsers, and also on any resolution.

**Comment by GawainLynch** [comment]
> Does your browser toolbar often look like this

Nope …

![image](https://user-images.githubusercontent.com/1427081/30816833-589eed00-a217-11e7-888f-cec595ebd998.png)

**Comment by GawainLynch** [comment]
Jokes aside, I don't see the harm

**Comment by bobdenotter** [comment]
@GawainLynch Yeah, my browser often looks like that, too. I agree that this suggestion seems harmless. For people like yourself (or often myself) it doesn't improve it, but it doesn't hurt either. 

Ideally we'd "SVG" an appropriate icon into the displayed "favicon", but that would require a lot of code. If we decide to do this, and people value it, we can always decide to put in the extra effort for that. :-)

**Comment by anketwokings** [comment]
Do we need emoji's on all the pages? Maybe we can use it on:
- pages you edit (pencil)
- settings pages (sprocket)

And not try and force it on the other pages. Maybe you might overshoot the clarity you are trying to achieve. 
Other than that, great idea!

**Comment by bobdenotter** [comment]
This is still on my planning to do, stale-bot. 🤖

**Comment by bobdenotter** [comment]
Still on the list for V4, stalebot.

**Comment by zomars** [comment]
Stalebot just won't give up!

**Comment by bobdenotter** [comment]
@zomars I've added the `keep` label now, so it won't happen again. ;-)

**Comment by Boorj** [comment]
from [here](https://github.com/bolt/bolt/issues/7641#issuecomment-417320965):
![image](https://user-images.githubusercontent.com/2145313/44854897-02fbe500-ac94-11e8-8672-eee487c3ee26.png)

**Comment by SvanteRichter** [comment]
IMO this still would be better in the favicon.
